tests/www_rbac/test_security.py

```diff
@@ -107,6 +107,20 @@ def test_init_role_modelview(self):
         self.assertIsNotNone(role)
         self.assertEqual(len(role_perms), len(role.permissions))
 
+    def test_update_and_verify_permission_role(self):
+        self.security_manager.init_role('User', [], [])
```

Comment:
Seems using `'User'` here break `test_log_success_for_user (tests.www_rbac.test_views.TestDagACLView)`.

May I suggest to use a "fake" role here for this test, say `self.security_manager.init_role('Test_Role', [], [])`, then make the same change for line 119 as well.

It should help pass the CI tests.

Reply:
Err... Seems this change I suggested didn't really help after your latest commit...

Reply:
even after the role_name change, test_log_success_for_user seems to be failing.